src/zzz_od/game_settings/game_settings_profile_service.py

```
@@ -0,0 +1,221 @@
+            self._close_game()
+        except GameSettingsProfileError as error:
+            errors.append(str(error))
+
+        try:
+            self._import_profile(normal_profile_path)
+            log.info("已恢复正常游戏画质配置")
+        except GameSettingsProfileError as error:
+            errors.append(str(error))
+
+        if errors:
+            raise GameSettingsProfileError("；".join(errors))
```

Ruff (0.16.1)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)